src/analyses/ai.h

```diff
@@ -9,6 +9,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_ANALYSES_AI_H
 #define CPROVER_ANALYSES_AI_H
 
+#include "util/make_unique.h"
```

Comment:
For consistency, please use `<util/make_unique.h>`, and place alongside the other includes from util/. Applies across this patch.